src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -829,6 +829,10 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                     return Failed;
                 }
             }
+            catch (IOException e)
+            {
+                MessageProvider.ReportIOException(e, consoleOutput);
```

Comment:
>MessageProvider.ReportIOException(e, consoleOutput); [](start = 16, length = 52)

It looks like this could simply swallow the exception without producing any output. Also, do we want to return ```Succeeded``` in this case?